runtime/Cargo.toml

```diff
@@ -59,6 +57,7 @@ name = "solana_runtime"
 
 [dev-dependencies]
 assert_matches = "1.5.0"
+ed25519-dalek = "=1.0.1"
```

Comment:
version looks to be remained same as intended. :)

Reply:
Not sure what you mean, is there an action item here?

Reply:
there is no action needed. my above comment is strictly just a random comment. I tend to write something verbosely. in this case, I left a comment to indicate i checked there's no accidental version bump of underlying crate (which could potentially create cluster incompatibility).